Custom highlighted fields (#3852)
elastic/security-docs: docs/detections/rules-ui-create.asciidoc

.. *False positive examples* (optional): List of common scenarios that may produce
false-positive alerts.
.. *MITRE ATT&CK^TM^ threats* (optional): Add relevant https://attack.mitre.org/[MITRE] framework tactics, techniques, and subtechniques.

> .. *Custom highlighted fields* (optional): Specify highlighted fields for personalized alert investigation flows. Fields with data are added to the [investigation-section,Highlighted fields] section within the alert details flyout. You can also find custom highlighted fields in the About section of the rule details page.

Reviewer: Is "Fields with data are added..." meant to imply that fields with no values wouldn't be added to these sections? If so, would it be beneficial to state this explicitly?

> .. *Custom highlighted fields* (optional): Specify highlighted fields for personalized alert investigation flows. Fields with values are added to the [investigation-section,Highlighted fields] section within the alert details flyout. Fields without values aren't added. After you create the rule, you can find all custom highlighted fields in the About section of the rule details page.

Author: ```suggestion
.. *Custom highlighted fields* (optional): Specify highlighted fields for personalized alert investigation flows. Fields with values are added to the <<investigation-section,Highlighted fields>> section within the alert details flyout. Fields without values aren't added. After you create the rule, you can find all custom highlighted fields in the About section of the rule details page.
```

+
NOTE: There's no limit to the number of custom highlighted fields you can add.  

> //Will change brackets to double angled brackets once merge PR #3816

Author: Removing comment because merged PR #3816.
```suggestion
```

.. *False positive examples* (optional): List of common scenarios that may produce
false-positive alerts.
.. *MITRE ATT&CK^TM^ threats* (optional): Add relevant https://attack.mitre.org/[MITRE] framework tactics, techniques, and subtechniques.
.. *Custom highlighted fields* (optional): Specify highlighted fields for personalized alert investigation flows. Fields with data are added to the [investigation-section,Highlighted fields] section within the alert details flyout. You can also find custom highlighted fields in the About section of the rule details page.
+

> NOTE: There's no maximum limit to the number of custom highlighted fields you can add.  

Author: ```suggestion
NOTE: There's no limit to the number of custom highlighted fields you can add.  
```

.. *False positive examples* (optional): List of common scenarios that may produce
false-positive alerts.
.. *MITRE ATT&CK^TM^ threats* (optional): Add relevant https://attack.mitre.org/[MITRE] framework tactics, techniques, and subtechniques.

> .. *Custom highlighted fields* (optional): Specify additional fields to display in the [investigation-section,Highlighted fields] section of the alert details flyout.

Author: ```suggestion
.. *Custom highlighted fields* (optional): Specify highlighted fields for personalized alert investigation flows. Fields with data are added to the [investigation-section,Highlighted fields] section within the alert details flyout. You can also find custom highlighted fields in the About section of the rule details page.
+
NOTE: There's no maximum limit to the number of custom highlighted fields you can add.  
```
